parm/config/gfs/config.resources

```diff
@@ -1030,6 +1030,153 @@ elif [[ ${step} = "gempak" ]]; then
     export memory_gempak="4GB"
     export memory_gempak_gfs="2GB"
 
+elif [[ ${step} = "mos_stn_prep" ]]; then
+
+    export wtime_mos_stn_prep="00:10:00"
+    export npe_mos_stn_prep=3
+    export npe_node_mos_stn_prep=3
+    export nth_mos_stn_prep=1
+    export memory_mos_stn_prep="5GB"
+    export NTASK=${npe_mos_stn_prep}
```

Comment:
I suggest add double quotes to `${npe_mos_stb_prep}`. This will appease the linter.

Reply:
Done, thanks @HenryWinterbottom-NOAA !

Reply:
Note, not going to add the double quotes around `${step}` since that isn't needed via shellcheck rules. Have added around the other new variables though. When I ran shellcheck on Orion for SC2086 it didn't flag those variables but perhaps it should have...so I have made the updates.

Reply:
As long as the linter is happy, I suppose that is all that matters. :-)

Thank you for addressing these. Most of them are likely overkill, but I am trying to avoid adding work for the future.